frontend/ui/widget/menu.lua

```diff
@@ -91,6 +91,7 @@ local MenuItem = InputContainer:extend{
     infont = "infont",
     linesize = Size.line.medium,
     single_line = false,
+    single_line_disabled = false, -- set to true to always use TextBoxWidget
     multilines_show_more_text = false,
```

Comment:
Or `force_multilines` or `multilines_forced` to stay positive and less cumbersome ? :)